test/rootfs/setup.sh

```diff
@@ -227,6 +227,13 @@ git clone https://github.com/sstephenson/bats.git "${tmpdir}/bats"
 curl -fsLo "/usr/local/bin/jq" "http://stedolan.github.io/jq/download/linux64/jq"
 chmod +x "/usr/local/bin/jq"
 
+# install protobuf compiler
+curl -sL https://github.com/google/protobuf/releases/download/v3.0.0/protoc-3.0.0-linux-x86_64.zip > ~/protoc.zip
+trap "rm ~/protoc.zip" EXIT
```

Comment:
I forgot that traps overwrite handlers. We also have umount and netns cleanup stuff attempting to be run on EXIT too, probably need hack together a solution to clean that up too.
For now I have changed it to use a single tmpdir atleast so we shouldn't leak files.